.github/workflows/federated-integ-test.yml

```diff
@@ -178,6 +178,17 @@ jobs:
         with:
           name: test-logs
           path: lte/gateway/logs.tar.gz
+      - name: Publish results to Firebase
+        if: always() && github.event.workflow_run.event == 'push'
+        env:
+          FIREBASE_SERVICE_CONFIG: ${{ secrets.FIREBASE_SERVICE_CONFIG }}
+          REPORT_FILENAME: "feg_integ_test_${{ env.SHA }}.html"
+        run: |
+          npm install -g xunit-viewer
```

Comment:
I think you are missing those steps: 
```
python -m pip install --upgrade pip
pip install Pyrebase
```

Reply:
@tmdzk got your point, but I don't see them in lte-integ-test.yml either. I took the sections from there, the whole process is quite similar.

Reply:
you're installing firebase_admin above so that's fine that's the only one you need